config/routes.rb

```diff
@@ -3,10 +3,11 @@
   devise_for :users
   root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show, :new, :create]
+    resources :posts, only: [:index, :show, :new, :create, :destroy]
   end
   resources :posts do
-    resources :comments, only: [:create, :new]
+    resources :comments, only: [:create, :new, :destroy]
     resources :likes, only: [:create, :destroy]
   end
+  delete 'posts/:id', to: 'posts#destroy', as: :destroy_post
 end
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on making the authorization so far :+1: but it would be nice if you follow the given URLs in the previous milestones :thinking: such as :arrow_down:

```ruby
1. /users
2. /users/id
3. /users/user_id/posts/
4. /users/user_id/posts/id
```